### PR TITLE
Game start with empty board

### PR DESCRIPTION
**Pull request by digitopolis**
When a new Game object is created, initialize with a Board property: an array of 9 empty strings (representing the spaces on the board)

### CODE REVIEW

**Comment by digitopolis** [comment]
[Game begins with an empty board](https://trello.com/c/BQeZ8Rp8/12-game-begins-with-an-empty-board)

**Review by serpient — APPROVED** [review]
Approved! Just clean up the repo as necessary with gitignore

File: TicTacToe/Game.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace TicTacToe
+{
+    public class Game
+    {
+        public Game()
+        {
+            this.Board = new string[] { "", "", "", "", "", "", "", "", "" };
```
Comment (yaoandrew) [inline TicTacToe/Game.cs:9]: `this.Board = new string [9];` ⬅️ does this do the same thing?

**Review by yaoandrew — APPROVED** [review]
LGTM :shipit:

File: TicTacToe/Game.cs (hunk 1 above)
Comment (digitopolis) [reply TicTacToe/Game.cs:9]: yes it does

**Pull request merged**